controller/engine_controller.go

```diff
@@ -187,6 +187,18 @@ func NewEngineController(
 	}
 	ec.cacheSyncs = append(ec.cacheSyncs, ds.InstanceManagerInformer.HasSynced)
 
+	if _, err = ds.PodInformer.AddEventHandlerWithResyncPeriod(cache.FilteringResourceEventHandler{
```

Comment:
Why do we need to sync on share manager pod change now? It doesn't look like the engine controller uses the share manager pod as an input, and I think for anything we need it to do in this PR (mostly just delete the engine instance), it should do it in response to a volume controller update to `desireState`.

Reply:
Engine need to quickly change ownership to a the same node as the RWX pod so perhaps this is the reason @james-munson want to watch RWX pod inside engine controller.

Though, I suspect that update to desireState could also trigger the change. Let me try to remove this informer to see if it break anything

Reply:
Removing this `PodInformer` doesn't seem to hurt anything. I am planing to remove it now. Is there any concern @james-munson ?

Reply:
OK, if you have tested and it doesn't seem to delay the sync by engine, then it certainly simplifies things.  We just don't want a lag on engine and replica handling after the pod is created.